workshops/personal_website/README.md

```diff
@@ -164,330 +130,221 @@ Every HTML file follows this template:
 </html>
 ```
 
-Let's put this template inside of our HTML file:
+This structure is common to all HTML pages. In fact, you can take a look for
+yourself! Just right click on any web page, including this one, and click "View
+source" to see what's going on behind the scenes. You'll find each of these
+elements on every page-- the doctype, and an HTML tag wrapped around a head and
+body.
+
+Before proceeding, we'll briefly go over what this template means. HTML works
+by storing information inside tags. `<html></html>` is an example of one such
+tag. Inside `<html></html>`, we've placed two other sets of tags:
+`<head></head>` (which wraps around the "head") and `<body></body>` (which
+wraps around the "body"). The body holds everything you would see in the actual
+tab/window when you open the page, while the head conveys information about the
+page to the browser.
+`<!DOCTYPE html>` tells the browser what version of HTML to expect. Since it is
+a language, HTML is constantly growing and updating, so there are multiple
+versions. In our case, we are going to use HTML5, the latest version.
+
+### 2) Previewing the Page
+
+Let's check out what our HTML file looks like in Live Preview! First, we'll
+save the file by clicking "File" → "Save" (or use the shortcut CTRL+s /
+Command+s). Then, preview what the website looks like by clicking "Preview" →
+"Live Preview File"
+
+![](img/c9_live_preview.gif)
+
+As you can see, the page is blank. This is because we haven't added anything to
+the `body` section yet. Let's add some content!
+
+### 3) Adding Text to the Body
+
+As mentioned before, all information is wrapped in tags. Tags are predefined in
+the language; think of them as the words in the language. For text, HTML
+provides a number of tags to store text. We'll be using two of the most basic
+ones: the h1 tag (`<h1>`) and the paragraph tag (`<p>`). The h1 tag is the
+first in a series of heading tags, with `h1` being the highest ranking, and
+`h6` being the lowest ranking. Just as with the other tags, you can place
```

Comment:
I do not want to associate the heading tags with size. Size is a look-and-feel thing, and should be handled with CSS. You could just as well make them all the same size, using CSS.